vaas-app/src/vaas/manager/api.py

```diff
@@ -69,6 +70,33 @@ class Meta:
             'cluster': ALL_WITH_RELATIONS
         }
 
+    def save_m2m(self, bundle):
+        logger = logging.getLogger('vaas')
```

Comment:
I'd rather move it to the module level after the imports, so it can be easily reused and catch errors earlier